workflow/labels.md

```diff
@@ -0,0 +1,55 @@
+# Labels descriptions
+
+In this part you will find a description of all the labels we use on issues.
+
+## Labels
+
+| Label       | Description                                              |
+| -----       | -----------                                              |
+| Don’t merge | Until this label is here the pull-request won’t be merge |
+| Duplicate   | This issue already exists                                |
+| Epic        |                                                          |
+| Invalid     | The maintainers don’t agree with the issue               |
+| Won’t fix   |                                                          |
+
+## Priority
+
+| Label    | Description                 |
+| ----     | ---                         |
+| Critical | Has to be fixed ASAP        |
+| High     | The issue needs to be fixed |
+| Medium   |                             |
+| Low      |                             |
+
+## Review
+
+Each review label has 3 steps:
+* **-1:** Not approved
+* **Needed:** needs to get reviewed
+* **+1:** Approved
+
+| Label  | Description                     |
+| ----   | ---                             |
+| Code   | Validation steps of your code   |
+| Design | Validation steps of your design |
+| QA     | Validation steps of QA          |
+| UX     | Validation steps of UX          |
+
+## Status
+
+| Label   | Description                                                     |
+| ----    | ---                                                             |
+| Blocked | The issue is block by another issue (put it in the description) |
+| Doing   | Actually working on the issue                                   |
+| Review  | The issue is in review                                          |
```

Comment:
I don't think we need doing or review labels anymore.

Reply:
I agree. This is the same as the approval on an issue. But I think we need it for the zenhub plugin?

Reply:
@tbille zenhub doesn't need any labels, it maintains its own state completely independently of labels.